bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -188,21 +195,24 @@
             .withDescription("Change the sound volume of a device")
             .withStateDescription(
                     new StateDescription(BigDecimal.ZERO, new BigDecimal(100), null, "%d %%", false, null))
-            .withCategory("SoundVolume").build();
+            .withCategory("SoundVolume")
+            .withTags(Arrays.asList(SystemTags.PROP_NOISE, SystemTags.CAP_DIMMABLE, SystemTags.CAP_VOLUME)).build();
```

Comment:
Wouldn't "Capability:Volume" not enough? Ok music is sometimes reffered to as "Noise" :) but I wouldn't take this as an argument here. Also "Capability:Dimmable" is more for lights, isn't it?

Reply:
Yes, "dimmable" is for light, I only put it here for starting this discussion. I was wondering how GUIs will detect whether to render a slider for such a channel... But they can query the ItemType for that...

Reply:
Hmm, we wanted to avoid querying the item type. I think, if it's volume (and not switchable - then it would be mute), a slider could always be rendered.